tests/expressions/tests.py

```diff
@@ -884,6 +885,17 @@ def test_range_lookup_allows_F_expressions_and_expressions_for_integers(self):
             ordered=False
         )
 
+    def test_range_lookup_allows_length_two_namedtuples(self):
+        EmployeeRange = namedtuple('EmployeeRange', ['minimum', 'maximum'])
+        self.assertQuerysetEqual(
+            Company.objects.filter(num_employees__range=EmployeeRange(minimum=1, maximum=100)),
+            [
+                '<Company: 5020 Ltd>', '<Company: 5040 Ltd>', '<Company: 5050 Ltd>',
+                '<Company: 5060 Ltd>', '<Company: 99300 Ltd>',
+            ],
+            ordered=False
+        )
```

Comment:
We want to get rid of `repr` in `assertQuerysetEqual()`, can we change to the `self.assertSequenceEqual()`, e.g.
```python
    @classmethod
    def setUpTestData(cls):
         ...
         cls.c5 = Company.objects.create(name='99300 Ltd', num_employees=99, num_chairs=300, ceo=ceo)

    def test_range_lookup_namedtuple(self):
        EmployeeRange = namedtuple('EmployeeRange', 'minimum maximum')
        qs = Company.objects.filter(
            num_employees__range=EmployeeRange(minimum=51, maximum=100),
         )
        self.assertSequenceEqual(qs, [self.c5])
```

Reply:
Ah yes, that big change.